### PR TITLE
arm/src/amebalite: temporary fix for tash issue

### PR DESCRIPTION
**Pull request by abhishek-samsung**
Increase uart irq priority to the highest as a temporary fix

Signed-off-by: Abhishek Akkabathula <a.akkabathul@samsung.com>

### CODE REVIEW

File: os/arch/arm/src/amebalite/amebalite_serial.c [hunk 1]
```diff
@@ -558,7 +558,7 @@ static int rtl8720e_log_uart_irq(void *Data)
 
 static int rtl8720e_log_up_attach(struct uart_dev_s *dev)
 {
-	InterruptRegister((IRQ_FUN)rtl8720e_log_uart_irq, RTL8720E_UART_LOG_IRQ-16, NULL, 4);
+	InterruptRegister((IRQ_FUN)rtl8720e_log_uart_irq, RTL8720E_UART_LOG_IRQ-16, NULL, 0);
```
Comment (thapav) [inline os/arch/arm/src/amebalite/amebalite_serial.c:561]: @fengyunlai COuld you please see why UART Rx has an issue with LOGUART (4) ?

Comment (thapav) [reply os/arch/arm/src/amebalite/amebalite_serial.c:561]: On latest loadable_apps configuration*

Comment (fengyunlai) [reply os/arch/arm/src/amebalite/amebalite_serial.c:561]: Hi Thapa, we have similar issue for our IPC irq, which we are debugging right now. (if we set a higher priority to IPC irq like you did for loguart irq, the issue is temporarily fixed also) Actually all these issues happen after we upgrade docker from 1.5.5 to 1.6.0. I will send an email to Sagam later today. Thank you.

**Comment by sunghan-chang** [comment]
@abhishek-samsung @thapav Could you confirm this is not necessary anymore due to PR #5619 ?

**Comment by sunghan-chang** [comment]
@abhishek-samsung @thapav If it is, let's close this PR.

**Comment by abhishek-samsung** [comment]
Closing this PR as reverting docker version to 1.5.5 + #5619 fixed the issue

**Pull request closed without merging**